web/js/components/timeline/timeline-data/timeline-data.js

```diff
@@ -393,6 +429,23 @@ function mapStateToProps(state) {
 }
 
 const mapDispatchToProps = (dispatch) => ({
+  onInfoClick: () => {
+    const key = 'DATA_PANEL_INFO_MODAL';
+    googleTagManager.pushEvent({
+      event: 'data_panel_info',
+    });
+    dispatch(
+      toggleCustomContent(key, {
+        headerText: 'Data Coverage Panel',
```

Comment:
In some places we call it the "layer coverage panel",  and the modal header calls it the "data coverage panel". Would be better to have them consistent

Reply:
Timeline Availability Panel - Is what we've used in articles and blog posts.

Reply:
I've revised the naming to Layer Coverage (Panel) throughout the components for consistency.